arangodb-net-standard/CollectionApi/GetCollectionsOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace ArangoDBNetStandard.CollectionApi
+{
+    public class GetCollectionsOptions
+    {
+        public bool? ExcludeSystem { get; set; }
+
+        public string ToQueryString()
+        {
+            List<string> query = new List<string>();
+            if (ExcludeSystem != null)
+            {
+                query.Add("excludeSystem=" + (ExcludeSystem.Value ? 1 : 0));
```

Comment:
From the API docs:
> By providing the optional query parameter excludeSystem with a value of true, all system collections will be excluded from the response.

So can we instead pass stringified boolean here?  e.g. `"excludeSystem=" + ExcludeSystem.ToString().ToLower()`